R/rmarkdown.R

```diff
@@ -0,0 +1,49 @@
+
+#' Add an RMarkdown Template
+#'
```

Comment:
Maybe a bullet list here briefly describing the files created?

I've also switch usethis to use roxygenmd, so can you please convert `\code{x}` to `` `x` ``

Then I think it's ready to merge

Reply:
Great. I've had a go at those changes.